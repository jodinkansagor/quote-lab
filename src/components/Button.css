@import url('https://fonts.googleapis.com/css?family=Caveat&display=swap');

button {
  font-family: 'Caveat', cursive;
  font-size: 20px;
  border-radius: 5px;
  margin-top: 15px;
  background-color: brown;
  color: beige;
  border: solid 0px white;
  border-radius: 5px;
  padding: 7px;
}

button:hover {
  background-color: beige;
  color: brown;
}

button:active {
  background-color: beige;
  color: brown;
}