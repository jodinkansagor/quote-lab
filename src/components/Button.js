import React from 'react';
import PropTypes from 'prop-types';

const Button = ({ onClick }) => (
  <button onClick={onClick}>Get Quote</button>
);

Button.propTypes = {
  onClick: PropTypes.func.isRequired
};

export default Button;
