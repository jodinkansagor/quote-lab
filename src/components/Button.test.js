import React from 'react';
import { shallow } from 'enzyme';
import Button from './Button';

describe('Button component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Button onClick={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
