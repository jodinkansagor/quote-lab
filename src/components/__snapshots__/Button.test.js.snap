// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component matches a snapshot 1`] = `
<button
  onClick={[Function]}
>
  Get Quote
</button>
`;
